scripts/qa_crowbarsetup.sh

```diff
@@ -5204,11 +5205,16 @@ function oncontroller_testpreupgrade_simple
             complain 11 "No tempest image found. Please make sure the tempest barclamp created its images."
         fi
     fi
-    openstack stack create upgrade_test -t \
-        /root/scripts/heat/2-instances-cinder.yaml \
-        --parameter image=$tempest_image $heat_stack_params
-    wait_for 15 20  "openstack stack list | grep upgrade_test | grep CREATE_COMPLETE" \
-                    "heat stack for upgrade tests to complete"
+    echo "starting $upgrade_test_stack_count test stack(s)..."
+    suffix=''
+    for i in $(seq 1 $upgrade_test_stack_count); do
+        openstack stack create "upgrade_test$suffix" -t \
+            /root/scripts/heat/2-instances-cinder.yaml \
+            --parameter image=$tempest_image --parameter suffix="$suffix" $heat_stack_params
+        wait_for 15 20 "openstack stack show \"upgrade_test$suffix\" | grep CREATE_COMPLETE" \
```

Comment:
Just a nit, but with Heat one could also use the CLI client's `--wait` option which will (1) poll until the stack is complete  and (2) output a running tally of resources as they are being completed.